cmd/ps.go

```diff
@@ -56,7 +56,9 @@ func PsScale(appID string, targets []string) error {
 			fmt.Printf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
```

Comment:
also the current implementation will cause ps:scale to return 0 (meaning success) when they call `deis ps:scale web=-1`. That's not really what we would want to do in this scenario